src/background/api-handlers.ts

```diff
@@ -7,7 +7,9 @@ function tweetFormData(target: FeedbackTarget): FormData {
 
   const status = target.feedbackState.editorState.getCurrentContent().getPlainText('\u0001')
   tweetData.append('status', status)
-  tweetData.append('domain', target.domain!)
+
+  const domain = target.feedbackState.twitterHandle.matchingDomain || target.parsedUrl!.host
```

Comment:
the matchingDomain clears along with the twitter handle so it makes sense to include it alongside the twitterHandle (see before where we previously had `isActualHandle`. Agreed that this looks ugly, but since this works and is well tested I'm going to put a TODO on it and we can touch it after launch.